pom.xml

```diff
@@ -452,5 +452,9 @@
     <project.httpcore.version>4.0.1</project.httpcore.version>
     <project.jdo2-api.version>2.3-eb</project.jdo2-api.version>
     <project.transaction-api.version>1.1</project.transaction-api.version>
+    <project.datanucleus-core.version>3.2.2</project.datanucleus-core.version>
```

Comment:
Should this be 3.2.1?

Reply:
Datanucleus versions their artifacts independently of each other. So not all versions of core have plugins at the same exact version. These were also the currently specified versions.

We can update separately if necessary, but newer versions have dropped support for older JVMs byte code manipulation.

Reply:
Ahh ok gotcha. SGTM.